core/src/main/java/com/linecorp/armeria/common/ClientCookieDecoder.java

```diff
@@ -230,9 +232,14 @@ private static void parse7(CookieBuilder builder, String header,
     }
 
     private static void parse8(CookieBuilder builder, String header,
-                               int nameStart) {
+                               int nameStart, int valueStart, int valueEnd) {
         if (header.regionMatches(true, nameStart, CookieHeaderNames.HTTPONLY, 0, 8)) {
             builder.httpOnly(true);
+        } else if (header.regionMatches(true, nameStart, CookieHeaderNames.SAMESITE, 0, 8)) {
+            final String sameSite = computeValue(header, valueStart, valueEnd);
+            if (isValidSameSite(sameSite)) {
+                builder.sameSite(sameSite);
```

Comment:
https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-07#section-4.1.2.7
> If the "SameSite" attribute's value is something other
   than these three known keywords, the attribute's value will be
   subject to a default enforcement mode that is equivalent to "Lax".

I guess we need to set `SameSite` as `Lax` if it's other than those three values.
Also how about using Enum for `SameSite` as Netty does?

Reply:
I see. I changed it to set `Lax` as default.

> Also how about using Enum for SameSite as Netty does?

I used Netty Enum to get `Lax` string here. Is this change correct for your suggestion ? 🤔

Reply:
I was thinking of forking the `SameSite` class to Armeria and using it as the return type.
(i.e. `SameSite sameSite() {...}`)
But I can't think of any good way to use it as the return type without a breaking change. 😅 
So let's leave it as is. We might consider using Enum from 2.0.

Last but not least, could you add a test case for the `sameSite`?

Reply:
>  let's leave it as is. We might consider using Enum from 2.0.

I see. Thanks for answering.  😄